### PR TITLE
Fake profile pr to test actions

### PR DESCRIPTION
**Pull request by starksm64**
Signed-off-by: Scott M Stark <starksm64@gmail.com>

## Specification XA template (Java Action Test)
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [x] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [ ] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [ ] Updated release record
   - [ ] Generated IP Log
   - [ ] Email to PMC
   - [ ] Start release review by emailing EMO 
- [ ] The URL of the OSSRH staging repository for the api, javadoc:
      <add URL here>
- [ ] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      <add URL here>
- [ ] The URL of the compatibility certification request issue:
      TBD-URL
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory. 
N/A

### CODE REVIEW

**Comment by starksm64** [comment]
# Spec Review Checklist

1. Spec PR
  - [x] (a) PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [x] (b) Directory of form {spec}/x.y
  - [ ] (c) PDF of form jakarta-{spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [ ] (d) HTML of form jakarta-{spec}-spec-x.y.html ("-spec" preferred but not required)
  - [ ] (e) Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [x] (f) Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [ ] (g) No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [ ] (h) Staging repository link of the form https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/{spec}/jakarta.{spec}-api/x.y.z/
  - [ ] (i) EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [ ] (j) Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [ ] (k) (Optional) Second PR for just apidocs

2. _index.md
  - [ ] Link to project release plan of the form https://projects.eclipse.org/projects/ee4j.{spec}/releases/x.y/plan
  - [ ] Link to spec pdf
  - [ ] Link to spec html
  - [ ] Link to apidocs
  - [ ] Link to final TCK download zip file of the form https://download.eclipse.org/jakartaee/{spec}/x.y/*{spec}-tck-x.y.z.zip (The folder path is required, the file name pattern is preferred.)
  - [ ] Link to API jar file of the form https://search.maven.org/artifact/jakarta.{spec}/jakarta.{spec}-api/x.y.z/jar
  - [ ] Name of and link to at least one Compatible Implementation

3. javadocs
  - [ ] Footer contains Eclipse copyright and link to license
  - [ ] ESFL license is included, usually as doc-files/speclicense.html
  - [ ] no META-INF directory in PR
  - [ ] javadocs-jar artifact matches apidocs (optional for this release)

4. Spec PDF
  - [ ] Correct spec title
  - [ ] Version number of the form x.y, not x.y.z
  - [ ] Correct Eclipse copyright line
  - [ ] No DRAFT or SNAPSHOT
  - [ ] Correct Logo

5. Spec HTML
  - [ ] Same as PDF

6. TCK zip file
  - [ ] README file (optional for this release)
  - [ ] EFTL license file, preferably named LICENSE.md
  - [ ] User's Guide (or equivalent documentation)
  - [ ] How to test the Compatible Implementation(s) listed in _index.md above with the TCK (may be in UG)

7. TCK User's Guide (or equivalent documentation)
  - [ ] Software requirements listed
  - [ ] Installation and configuration described
  - [ ] How to run tests
  - [ ] Where to file challenges

8. Compatibility certification request
  - [ ] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
  - [ ] SHA-256 fingerprint matches staged TCK zip file
  - [ ] Request issue has `certification` label.

9. TCK results summary
  - [ ] Page is hosted by Compatible Implementation project
  - [ ] Includes all information from certification request
  - [ ] Summary includes number of tests passed, failed, errors
  - [ ] SHA-256 fingerprint matches staged TCK zip file on cert request

10. If a Release Review is required, the specification project team contacts the EMO to initiate the review by sending an email to emo@eclipse.org.
    (A Release Review is not required if the current release is a Service Release based on a previously successful Major or Minor
    release as indicated by a release record on the project's Releases page, e.g., the [Jakarta Servlet releases page](https://projects.eclipse.org/projects/ee4j.servlet/reviews).)
    - [ ] An issue will be created by the EMO to track the release review.
    - [ ] The specification project team requests approval for the release from the PMC by sending an email to ee4j-pmc@eclipse.org.
    - [ ] The specification project team then delivers an IP Log to the IP Team for their review as described in https://www.eclipse.org/projects/handbook/#pmi-commands-iplog.

11. Update Jakarta EE API jar
  - [ ] Update the Jakarta EE API jar by submitting a PR to the jakartaee-api project that updates the version number of your API jar file.

**Review by starksm64 — COMMENTED** [review]
# Spec Review Checklist

1. Spec PR
  - [x] (a) PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
  - [x] (b) Directory of form {spec}/x.y
  - [ ] (c) PDF of form jakarta-{spec}-spec-x.y.pdf ("-spec" preferred but not required)
  - [ ] (d) HTML of form jakarta-{spec}-spec-x.y.html ("-spec" preferred but not required)
  - [ ] (e) Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
  - [x] (f) Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
  - [ ] (g) No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
  - [ ] (h) Staging repository link of the form https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/{spec}/jakarta.{spec}-api/x.y.z/
  - [ ] (i) EFTL TCK link of the form http://download.eclipse.org/.../+.zip
  - [ ] (j) Compatibility certification link of the form https://github.com/eclipse-ee4j/{project}/#{issue}
  - [ ] (k) (Optional) Second PR for just apidocs

File: coreprofile/10/_index.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ The goals for this release are:
 1. To provide a profile that contains a set of Jakarta EE Specifications targeting smaller runtimes suitable for microservices and ahead-of-time compilation.
 2. The TCK(s) for the Core profile should be possible to run independently of the Full Platform TCK.
 3. Java SE version will be aligned with the Platform Project's decision regarding Java SE versions.
+4. Some fake stretch goal, expanded 3...
```
Comment (scottmarlow) [inline coreprofile/10/_index.md:12]: action time?

**Review by scottmarlow — CHANGES_REQUESTED** [review]

**Comment by starksm64** [comment]
# Spec Review Checklist

**Pull request closed without merging**